wp-content/themes/core/components/header/subheader/css/subheader.pcss

```diff
@@ -0,0 +1,56 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Subheader
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-subheader {
+	position: relative;
+	display: flex;
+	align-items: center;
+	min-height: 260px;
+	background: var(--color-grey-light);
+	padding:  var(--spacer-20) 0;
+	color: var(--color-text);
```

Comment:
Text color shouldn't be necessary here. It would be applied to the text components (headings, paragraphs, etc) added inside the subheader rather than here.